src/renderer/families/polkadot/operationDetails.js

```diff
@@ -0,0 +1,364 @@
+/* eslint-disable consistent-return */
+// @flow
+import startCase from "lodash/startCase";
+import React, { useMemo } from "react";
+
+import { BigNumber } from "bignumber.js";
+
+import type { Currency, Unit, Operation, Account } from "@ledgerhq/live-common/lib/types";
+
+import { getDefaultExplorerView, getAddressExplorer } from "@ledgerhq/live-common/lib/explorers";
+
+import { openURL } from "~/renderer/linking";
+
+import {
+  OpDetailsTitle,
+  Address,
+  OpDetailsData,
+  OpDetailsVoteData,
+} from "~/renderer/modals/OperationDetails/styledComponents";
+import { Trans } from "react-i18next";
+import Box from "~/renderer/components/Box/Box";
+import { usePolkadotPreloadData } from "@ledgerhq/live-common/lib/families/polkadot/react";
+import Text from "~/renderer/components/Text";
+import FormattedVal from "~/renderer/components/FormattedVal";
+import CounterValue from "~/renderer/components/CounterValue";
+import { useDiscreetMode } from "~/renderer/components/Discreet";
+
+const helpURL = "https://support.ledger.com/hc/en-us/articles/FIXME";
+
+function getURLFeesInfo(op: Operation): ?string {
+  if (op.fee.gt(200000)) {
+    return helpURL;
+  }
+}
+
+function getURLWhatIsThis(op: Operation): ?string {
+  if (op.type !== "IN" && op.type !== "OUT") {
+    return helpURL;
+  }
+}
+
+function formatPalletMethod(palletMethod: ?string): string {
+  if (!palletMethod) return "";
+
+  return palletMethod
+    .split(".")
+    .map(startCase)
+    .join(" - ");
+}
+
+const redirectAddress = (currency: Currency, address: string) => () => {
```

Comment:
Yup that could be a good idea to add this DRY improvement on your board as a maintenance/refacto task.

In fact, this could even be improved by letting user choose for each coin the Explorer they want to use.
Providing the hook at the same time could DRY up the other coins too (this is a cross-coins & component issue).